src/internal_effect_untraced/query.ts

```diff
@@ -938,31 +938,31 @@ export const sandboxWith = Debug.untracedDual<
 
 /** @internal */
 export const service = Debug.methodWithTrace((trace) =>
```

Comment:
While we are at it, we can delete these methods as they are redundant. The following should work since `Query` is also a subtype of `Effect`.

```ts
Query.flatMap(<Tag>, (service) => ...)
```

Reply:
Removed all the `service*` functions in https://github.com/Effect-TS/query/pull/449/commits/090d6865b7b26f42df652f7727890cfb0f75b2ee. Any others you wanted to remove?